docs/src/add-services/varnish.md

```diff
@@ -34,11 +34,11 @@ You can define `<RELATIONSHIP_NAME>` as you like.
 `<APP_NAME>` should match the name you gave your app in your [app configuration](../create-apps/app-reference.md).
 
 The `configuration` block must reference a VCL file (`config.vcl` in this example).
-The file name is relative to the `.platform` directory.
+The file must be inside the `.platform` directory and is defined relative to that directory.
```

Comment:
Hmm. I personally find your suggestion more complex. I think because "reference to the file" seems somewhat unclear to me, but I'm not sure. 

I could try moving part of it to the previous sentence and using `path` to make it clear what needs to be relative. See if that helps.

Reply:
Yes 😻 That's much clearer and straightforward! Thanks!